.github/ISSUE_TEMPLATE/template-feature.yml

```diff
@@ -0,0 +1,31 @@
+name: New Feature
+description: Suggest an idea for this project
+title: "[Feature request]: "
+labels: ["Engine: Independent", "Priority: Medium", "OS: Independent", "Status: Available", "Type: Rrequest"]
```

Comment:
Is it compatible with the new labels?

Reply:
nope, it should be fixed by me in few days)

branch is outdated now and need to be updated